apps/frontend/.env.local.example

```
@@ -0,0 +1,22 @@
+# Firebase Client SDK (obtain from Firebase Console → Project Settings → Your Apps)
+NEXT_PUBLIC_FIREBASE_API_KEY=#your_api_key
+NEXT_PUBLIC_FIREBASE_AUTH_DOMAIN=#your_project.firebaseapp.com
+NEXT_PUBLIC_FIREBASE_PROJECT_ID=#your_project_id
+NEXT_PUBLIC_FIREBASE_STORAGE_BUCKET=#your_project.appspot.com
+NEXT_PUBLIC_FIREBASE_MESSAGING_SENDER_ID=#your_sender_id
+NEXT_PUBLIC_FIREBASE_APP_ID=#your_app_id
+
+# TrustlessWork API
+NEXT_PUBLIC_API_URL=https://api.trustlesswork.com
+NEXT_PUBLIC_API_KEY=your_trustlesswork_api_key
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_API_KEY key should go before the NEXT_PUBLIC_API_URL key

(UnorderedKey)